src/style.css

```
@@ -0,0 +1,94 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
+  color: rgb(255 255 255);
+  line-height: 300px;
+  text-align: center;
+}
+
+.block-1 {
+  background-color: rgb(255 0 0);
+}
+
+.block-2 {
+  background-color: rgb(204 0 0);
+}
+
+.block-3 {
+  background-color: rgb(153 0 0);
+}
+
+.block-4 {
+  background-color: rgb(102 0 0);
+}
+
+.block-5 {
+  background-color: rgb(51 0 0);
+}
+
+.block-6 {
+  background-color: rgb(0 0 0);
+}
+
+@media screen and (min-width: 900px) {
+  .block {
+    flex-basis: 33%;
+  }
+}
+
+@media screen and (min-width: 600px) {
+  .block-3 {
+    order: 4;
+  }
+}
+
+@media screen and (min-width: 600px) {
+  .block-4 {
+    order: 3;
+  }
+}
+
+@media screen and (min-width: 600px) {
+  .block-5 {
+    order: 4;
+  }
+}
```

You shouldn't add the same media queries
```suggestion
@media screen and (min-width: 600px) {
  .block-3 {
    order: 4;
  }
  
  .block-4 {
    order: 3;
  }
  
  .block-5 {
    order: 4;
  }
}
```